packages/lesswrong/components/onboarding/OnboardingFlow.tsx

```diff
@@ -38,7 +38,7 @@ const OnboardingFlow = ({stages, viewAsAdmin, classes}: {
 
   const {BlurredBackgroundModal} = Components
   return (
-    <BlurredBackgroundModal open className={classes.root}>
+    <BlurredBackgroundModal open className={classes.root} data-testid="onboarding-flow">
```

Comment:
```suggestion
    <BlurredBackgroundModal open className={classes.root} testId="onboarding-flow">
```
Should this be like this? Seeing as `testId` is a regular prop that gets set as a `data-testid` inside that component?